docs/developer-guide/base-maps/using-with-other-maps.md

```diff
@@ -0,0 +1,22 @@
+# Using deck.gl with other basemap services
```

Comment:
Merge this with the mapbox document since it's relying on mapbox-gl?

Reply:
done